src/components/async-route.vue

```diff
@@ -12,25 +12,27 @@ except according to the terms contained in the LICENSE file.
 <template>
   <div v-if="loading === 'tab'">
     <loading :state="showsLoading"/>
-    <component :is="component" v-if="component != null" :key="k" v-bind="props"
-      v-on="$listeners"/>
+    <component :is="component" v-if="component != null" :key="k"
+      v-bind="propsAndAttrs"/>
   </div>
   <page-body v-else-if="showsLoading">
     <loading :state="true"/>
   </page-body>
   <component :is="component" v-else-if="component != null" :key="k"
-    v-bind="props" v-on="$listeners"/>
+    v-bind="propsAndAttrs"/>
 </template>
 
 <script>
 import Loading from './loading.vue';
 import PageBody from './page/body.vue';
+
 import { loadAsync, loadedAsync } from '../util/async-components';
 import { noop } from '../util/util';
 
 export default {
   name: 'AsyncRoute',
   components: { Loading, PageBody },
+  inheritAttrs: false,
```

Comment:
I was second-guessing myself, but it looks like it does need to be `false`. Otherwise the attributes will be set on the root element of the component (which we don't want here).